### ISSUE TITLE
PhpDoc FQCN vs namespaces

### ISSUE THREAD

**Issue by hacfi** [stage1/docker-php#110]
Working on some PRs I noticed that the PhpDocs always use the full FQCN (see https://github.com/stage1/docker-php/blob/master/src/Docker/Docker.php#L30)

Example:
https://github.com/stage1/docker-php/blob/master/src/Docker/Docker.php#L30
```
    /**
     * @var \Docker\Manager\ImageManager
     */
    private $imageManager;
```
instead of just `@var ImageManager`.

Do you want to keep this or can I create a PR getting rid of the unnecessary long FQCN names? If so, should I add the use statements for classes which aren’t used in the PHP code but just in the PhpDocs?

See http://www.phpdoc.org/docs/latest/guides/types.html#class-name that this is valid PhpDoc.

**Comment by ubermuda** [comment]
Well, I'm not sure about that one. The idea to add `use` statements just for the phpdoc annoys me, but having `@var ImageManager` without the proper `use` statement would make it useless, so I'd vote for keeping the FQCN in the phpdocs.

I could easily be convinced otherwise with proper arguments, though.

**Comment by hacfi** [comment]
I personally don’t add use statements just for phpdoc but otherwise I’d prefer using the namespaced name. Something like https://github.com/stage1/docker-php/blob/master/src/Docker/Manager/ContainerManager.php#L178-185 would be a little easier to read. By the way I just realized that in several cases we already use the namespaced name. Will work on a PR WITHOUT use statements for classes which aren’t used in the php code.

**Comment by ubermuda** [comment]
I guess using the class name for classes that are already `use`d and the FQCN otherwise would be good yes

**Comment by hacfi** [comment]
Yes, will do that.

**Comment by joelwurtz** [comment]
closed in #147
